### ISSUE TITLE
Can i get a list of debloat apps?

### ISSUE THREAD

**Issue by vannessqn** [realKarthikNair/realme-ui-debloater#3]
Also is this include weather service? I read that uninstall it make the device go boot loop (scary). I have the realme Q3S android 11 with stock rom so from China so do you think something wrong can happen?

**Comment by realKarthikNair** [comment]
I am aware of the bootloop issue with weather service, so **I DID NOT INCLUDE IT**
To be on an even safer side, I didn't include weather app either (though it is safe to be removed, only removing weather service bootloops)

As of now, I am unable to get you the list of apps in the script since I didn't note it down myself while typing package names but most of them are self explanatory! The rest you can always look up into a search engine like Google. 

I'll definetly include the names of those apps in a later point of time, right now I am a bit busy with academical stuff

Hope this helps

**Comment by makkarankush68** [comment]
yeah that would be really helpful and incorporate a sense of credibility knowing what's been uninstalled.

**Comment by realKarthikNair** [comment]
1. I no longer use realme UI
2. I lost interest in this repo

so closing
